<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/newPrefix/image.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="predictedadjclose_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Predicted Adj. Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adjopenvsadjclose_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Adj.Open vs Adj.Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="highvslow_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>High vs Low</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="volume_pushButton">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Ubuntu Condensed&quot;;
color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Ubuntu Condensed&quot;;
border-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Select a graph to plot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
